todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/network/ApisUtils.java

```diff
@@ -0,0 +1,14 @@
+package com.example.android.architecture.blueprints.todoapp.network;
+
+/**
+ * Created by Sa7r on 6/18/2017.
+ */
+
+public class ApisUtils {
+    private static String BASE_URL="https://jsonplaceholder.typicode.com/";
```

Comment:
URLs and API keys should be stored securely and configured through flavors/build variants.